src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1351,9 +1402,60 @@ private void getVars(CORINFO_METHOD_STRUCT_* ftn, ref uint cVars, ILVarInfo** va
             // Just tell the JIT to extend everything.
             extendOthers = true;
         }
+
         private void setVars(CORINFO_METHOD_STRUCT_* ftn, uint cVars, NativeVarInfo* vars)
         {
-            // TODO: Debugging
+            if (_localSlotToInfoMap == null && _parameterIndexToNameMap == null)
```

Comment:
Depending on static/instance methods, the number of parameters are different which will potentially shift the var numbers (# of param + slot #) for locals. I just want to confirm the information from Jit and the information from IL PDB reader align them with respect to it  so that we don't map incorrect local/param data.